README.md

```diff
@@ -5,13 +5,25 @@ Chat bot for StackOverflow. Uses [amphp](https://github.com/amphp) libraries for
 
 [![Build Status](https://travis-ci.org/Room-11/Jeeves.svg?branch=master)](https://travis-ci.org/Room-11/Jeeves)
 
+## Requirements
```

Comment:
I for one ~~~welcome our new README overlords~~~ always check the README first for requirements.

It means having to maintain it twice but I personally can live with that.